app/client/models/ToggleEnterpriseModel.ts

```diff
@@ -0,0 +1,23 @@
+import {AppModel, getHomeUrl} from 'app/client/models/AppModel';
+import {bundleChanges, Disposable, Observable} from "grainjs";
+import {ConfigAPI, ConfigAPIImpl} from 'app/common/ConfigAPI';
+
+export class ToggleEnterpriseModel extends Disposable {
+  public readonly edition: Observable<string | null> = Observable.create(this, null);
+  private readonly _configAPI: ConfigAPI = new ConfigAPIImpl(getHomeUrl());
+  constructor (_appModel: AppModel) {
+    super();
+  }
+
+  public async fetchEnterpriseToggle(): Promise<void> {
+    const edition = await this._configAPI.getValue('edition');
+    bundleChanges(() => {
```

Comment:
Cargo-culted from [this](https://github.com/gristlabs/grist-core/blob/main/app/client/models/TelemetryModel.ts#L23).

Can you please explain why `TelemetryModel` does this? Maybe because it modifies a whole object, not just one boolean?

Reply:
It looks like a leftover. Maybe it did something more and noone cleared that. It isn't needed there.